pkg/cluster/clusterserviceprincipal_test.go

```diff
@@ -438,3 +458,44 @@ func TestUpdateOpenShiftSecret(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
See linked issue kubernetes/client-go#1184 for more information on why this is required. Potential upstream contribution if someone wants to tackle it.